### ISSUE TITLE
Tabs has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1930]
- id attribute value must be unique (issue with headings)

**Comment by redallen** [comment]
Closing in favor of #2158.
